### DISCUSSION TITLE
Chnage Climate Entity by automation

### DISCUSSION THREAD

**Discussion by theoldphilip** [Blackymas/NSPanel_HA_Blueprint#885] [Q&A]
Hi,

is it possible to change the climate entity used by the panel via an automation?

I have two virtual thermostats, one for cooling and one for heating. I would like to be able to control it on the panel when the underfloor heating is running in cooling mode.

Greeting Philip

**Comment by Doughboy68** [comment]
That is kinda what I asked... but I learned you can put a climate entity on any button and works.

https://github.com/Blackymas/NSPanel_HA_Blueprint/discussions/877#discussioncomment-6227407

Interesting idea to able to change the automation blueprint somehow in Home Assistant side of things... I don't think this is possible within HA itself.

**Comment by theoldphilip** [comment]
Thanks for the tip. I didn't know either :)

At least that's a working workaround for me.
Nevertheless, it would be nice if you could automate the whole thing :-).

**Comment by edwardtfn** [comment]
Well, this is a different case compared to #877.

There are some custom integrations ([like this one](https://github.com/jcwillox/hass-template-climate)) which allows creating template climate entities. You can use it to have a new climate entity which depending of the conditions you define will be pointing to your heater or to your cooler, then you can assign this new entity to your panel and you will have what you are looking for.
